### PR TITLE
[test] update kubeflow script

### PR DESCRIPTION
**Pull request by tejaschumbalkar**
*Description:*
- Add AWS region environment variable required during installation of kubeflow by kfctl client.
- EC2 instance do not have permissions to move the kfctl executable to /usr/local/bin dir, adding sudo permissions.

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license. I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: test/dlc_tests/eks/eks_manifest_templates/kubeflow/install_kubeflow_custom_kfctl.sh [hunk 1]
```diff
@@ -4,13 +4,13 @@ set -e
...
-    S3_BUCKET="kubeflow-kfctl-binary"
+    S3_URL="https://kubeflow-kfctl-binary.s3-us-west-2.amazonaws.com/kfctl_v1.0.2-1-g93e95e1_linux.tar.gz"
 
     if ! command -v kfctl &> /dev/null
     then
-        aws s3 cp s3://${S3_BUCKET}/kfctl_${KFCTL_VERSION}-1-g93e95e1_linux.tar.gz /tmp/kfctl_${KFCTL_VERSION}_linux.tar.gz
+        wget -O /tmp/kfctl_${KFCTL_VERSION}_linux.tar.gz ${S3_URL}
         tar -xvf /tmp/kfctl_${KFCTL_VERSION}_linux.tar.gz -C /tmp --strip-components=1
-        mv /tmp/kfctl /usr/local/bin
+        sudo mv /tmp/kfctl /usr/local/bin
```
Comment (tejaschumbalkar) [inline test/dlc_tests/eks/eks_manifest_templates/kubeflow/install_kubeflow_custom_kfctl.sh:13]: This script will be running as a part of CDK (lambda function) and `sudo` is required while running on EC2 instance.
Note: The PR builds will have failures as sudo is not supported.

File: test/dlc_tests/eks/eks_manifest_templates/kubeflow/install_kubeflow_custom_kfctl.sh [hunk 2]
```diff
@@ -4,13 +4,13 @@ set -e
 install_kfctl(){
     #install kfctl cli
     KFCTL_VERSION="v1.0.2"
-    S3_BUCKET="kubeflow-kfctl-binary"
+    S3_URL="https://kubeflow-kfctl-binary.s3-us-west-2.amazonaws.com/kfctl_v1.0.2-1-g93e95e1_linux.tar.gz"
```
Comment (arjkesh) [inline test/dlc_tests/eks/eks_manifest_templates/kubeflow/install_kubeflow_custom_kfctl.sh:7]: nit: why is this s3 link regionalized and the kfctl hard coded?

**Review by arjkesh — APPROVED** [review]

File: test/dlc_tests/eks/eks_manifest_templates/kubeflow/install_kubeflow_custom_kfctl.sh (hunk 2 above)
Comment (tejaschumbalkar) [reply test/dlc_tests/eks/eks_manifest_templates/kubeflow/install_kubeflow_custom_kfctl.sh:7]: S3 bucket are regionalized. kfctl binary was provided to overcome https://github.com/kubeflow/kubeflow/issues/5245

**Comment by tejaschumbalkar** [comment]
The issues on kubeflow/kubeflow#5245 has been addressed by the pr kubeflow/kfctl#401. 
Next steps:
1. Test the kubeflow script (install_kubeflow.sh) locally and verify that the kubeflow installation is successful in AWS network.
2. If the test are successful, point back the kubeflow setup script from install_kubeflow_custom_kfctl.sh file to install_kubeflow.sh file.

**Review by saimidu — APPROVED** [review]

**Pull request merged**